src/main/java/mate/controller/car/AddDriverToCarController.java

```diff
@@ -27,11 +28,11 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
     @Override
     protected void doPost(HttpServletRequest req, HttpServletResponse resp)
             throws IOException, ServletException {
-        long driverId = Long.parseLong(req.getParameter("driver_id"));
+        long driverId = Long.parseLong(req.getParameter(DRIVER_ID));
         long carId = Long.parseLong(req.getParameter("car_id"));
         Driver driver = driverService.get(driverId);
         Car car = carService.get(carId);
         carService.addDriverToCar(driver, car);
-        resp.sendRedirect("/cars/drivers/add");
+        resp.sendRedirect("/cars/drivers");
```

Comment:
Do you have servlet mapping to this url?

Reply:
fixed

Reply:
still can't find servlet responsible for processing request on this url ("/cars/drivers")